docs/integrations/snippets.md

```diff
@@ -0,0 +1,421 @@
+# Integration Snippets
+
+With the purpose of helping the development of new integrations, we have created some code "snippets" that can be used into new or existing integrations. If you are using VSCode as source code editor, please follow the next guide to install and use the available snippets https://github.com/JupiterOne/integration-engineering-toolbox/tree/main/integration-code-snippets.
```

Comment:
The snippets in the `integration-engineering-toolbox` repository are private. I'd recommend to work on a VSCode extension so we can add the steps to install that extension here. The other option would be to move those snippets to other public repository. @VDubber